roles/openshift_repos/vars/main.yml

```diff
@@ -1,2 +1,3 @@
 ---
 known_openshift_deployment_types: ['origin', 'online', 'enterprise']
+known_openshift_product_types: ['openshift', 'atomic-enterprise']
```

Comment:
@detiber  OSE and AE are both derived from origin so that is same for both as I understand.